Packs/ipinfo/Integrations/ipinfo_v2/ipinfo_v2.py

```diff
@@ -0,0 +1,197 @@
+        if 'asn' in raw_result:
+            asn = demisto.get(raw_result, 'asn.asn')
+            as_owner = demisto.get(raw_result, 'asn.name')
+            as_domain = demisto.get(raw_result, 'asn.domain')
+
+            if as_domain:
+                feed_related_indicators.append(Common.FeedRelatedIndicators(as_domain, FeedIndicatorType.Domain))
+
+        elif 'org' in raw_result:
+            org = raw_result.get('org', '')
+            org_parts = org.split(' ')
+            if ' ' in org:
+                asn, as_owner = org_parts[0], ' '.join(org_parts[1:])
+
+        organization = {
+            'Name': demisto.get(raw_result, 'company.name'),
+            'Type': demisto.get(raw_result, 'company.type')
+        } if 'company' in raw_result else None
+
+        company_domain = demisto.get(raw_result, 'company.domain')
+        if company_domain is not None:
+            feed_related_indicators.append(Common.FeedRelatedIndicators(company_domain, FeedIndicatorType.Domain))
+
+        abuse = {
+            'Address': demisto.get(raw_result, 'abuse.address'),
+            'Country': demisto.get(raw_result, 'abuse.country'),
+            'Name': demisto.get(raw_result, 'abuse.name'),
+            'Network': demisto.get(raw_result, 'abuse.network'),
+            'Phone': demisto.get(raw_result, 'abuse.phone'),
+            'Email': demisto.get(raw_result, 'abuse.email')
+        } if 'abuse' in raw_result else None
+
+        tags = []
+        for (tag_path, tag_name) in (('privacy.hosting', 'hosting'),
+                                     ('privacy.proxy', 'proxy'),
+                                     ('privacy.tor', 'tor'),
+                                     ('privacy.vpn', 'vpn')):
+            if demisto.get(raw_result, tag_path):
+                tags.append(tag_name)
+
+        city = raw_result.get('city')
+        region = raw_result.get('region')
+        postal = raw_result.get('postal')
+        country = raw_result.get('country')
+
+        description = ', '.join(filter(None, [city, region, postal, country]))
+
+        # parses geolocation
+        lat = lon = ''
+        loc = raw_result.get('loc', '')  # empty string as default on purpose,
+        if ',' in loc:
+            coordinates = loc.split(',')
+            lat, lon = float(coordinates[0]), float(coordinates[1])
+
+        entry_context = {'Address': raw_result.get('ip'),
+                         'Hostname': hostname,
+                         'ASN': asn,
+                         'ASOwner': as_owner,
+                         'Tags': tags,
+                         'Organization': organization,
+                         'Geo': {'Location': loc, 'Country': country, 'Description': description},
+                         'Registrar': {'Abuse': abuse} if abuse else None}
+
+        outputs_key_field = 'Address'  # marks the ip address
+
+        indicator = Common.IP(
+            ip=ip,
+            dbot_score=Common.DBotScore(indicator='ip',
```

Comment:
I believe this should be the indicator value.
```suggestion
            dbot_score=Common.DBotScore(indicator=ip,
```